faunadb/errors.py

```diff
@@ -18,6 +20,29 @@ class FaunaError(Exception):
   For documentation of error types, see the `docs <https://faunadb.com/documentation#errors>`__.
   """
 
+  @staticmethod
+  def from_status_code(code, response_dict):
+    # pylint: disable=no-member
+    if 200 <= code <= 299:
```

Comment:
I changed the way it worked. FaunaError.from_response now returns a FaunaError and doesn't raise.